packages/astro/components/ViewTransitions.astro

```diff
@@ -50,6 +50,14 @@ const { fallback = 'animate' } = Astro.props;
 		return el.dataset.astroReload !== undefined;
 	}
 
+	function isCrossOrigin(action?: string): boolean {
```

Comment:
We try to keep the file size of ViewTransition.astro small. Could you please inline that call?